cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/CassandraCradleStorage.java

```diff
@@ -336,18 +327,37 @@ protected void doStoreTestEventMessagesLink(StoredTestEventId eventId, StoredTes
 			throw new IOException("Error while storing link between test event "+eventId+" and "+messageIds.size()+" message(s)", e);
 		}
 	}
-	
+
 	@Override
 	protected CompletableFuture<Void> doStoreTestEventMessagesLinkAsync(StoredTestEventId eventId,
-			StoredTestEventId batchId, Collection<StoredMessageId> messageIds)
+																		StoredTestEventId batchId, Collection<StoredMessageId> messageIds)
 	{
 		List<String> messageIdsStrings = messageIds.stream().map(StoredMessageId::toString).collect(toList());
 		String eventIdString = eventId.toString();
-		return CompletableFuture.allOf(storeMessagesOfTestEvent(eventIdString, messageIdsStrings),
-				storeTestEventOfMessages(messageIdsStrings, eventIdString, batchId));

```

Comment:
Removal of `storeTestEventOfMessages` is not part of this task, I think.

Reply:
It was in with @Nikita-Smirnov-Exactpro in order to implement the linking of messages with their events. I split this method into two methods (added doStoreMessageTestEventLinkAsync(eventId, batchId, messageIds)).